spec/swagger_helper.rb

```diff
@@ -10,7 +10,7 @@
       info: {
         title: "API documentation",
         version: "v1",
-        description: "Auto generated doc",
+        description: "Auto generated document, run `bundle exec rake rswag` to regenerate",
```

Comment:
This is normally a description of the API to help API clients understand the purpose of the API rather than how to manage the API

Reply:
Good spot, yes that's not too helpful is it.

![Selection_20210512-01-655fd655fd655fd](https://user-images.githubusercontent.com/19378/117993332-8b811d80-b337-11eb-8ead-4a901d99474c.png)

@mattonfoot  I'll do something simple for now but if you have suggestions for something better let us know or do a PR when this is in. As I understand it this swagger UI will be seen by lead provider tech people.